android/src/main/java/io/ably/flutter/plugin/AblyMessageCodec.java

```diff
@@ -181,9 +188,9 @@ protected void writeValue(ByteArrayOutputStream stream, Object value) {
 
   private void WriteJsonElement(ByteArrayOutputStream stream, JsonElement value) {
     if (value instanceof JsonObject) {
-      super.writeValue(stream, (new Gson()).fromJson(value, Map.class));
+      super.writeValue(stream, (gson).fromJson(value, Map.class));
```

Comment:
Is there a need for the paranthesis now?

Reply:
removed: f7be2e8595b6e13b5e448f9030390e10247cfb84